app/service/auth_svc.py

```diff
@@ -1,19 +1,23 @@
 import base64
+import logging
 from collections import namedtuple
 
 from aiohttp import web, web_request
 from aiohttp.web_exceptions import HTTPUnauthorized, HTTPForbidden
+from aiohttp_jinja2 import render_template
 from aiohttp_security import api as aiohttp_security_api
 from aiohttp_security import SessionIdentityPolicy, check_permission, remember, forget
 from aiohttp_security import setup as setup_security
 from aiohttp_security.abc import AbstractAuthorizationPolicy
 from aiohttp_session import setup as setup_session
 from aiohttp_session.cookie_storage import EncryptedCookieStorage
 from cryptography import fernet
+from importlib import import_module
```

Comment:
nit: this is a standard library import and should follow the `from collections ...` import above.